src/AppBundle/Command/SendMailToProcurationProxiesCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Entity\ProcurationRequest;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class SendMailToProcurationProxiesCommand extends ContainerAwareCommand
+{
+    private $em;
+    private $mailjet;
+    private $messageFactory;
+
+    protected function configure()
+    {
+        $this
+            ->setName('app:mail:send-reminder-to-procuration-proxy ')
+            ->setDescription('Send a reminder, 48h after the match, to the procuration proxies.')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->em = $this->getContainer()->get('doctrine.orm.entity_manager');
+        $this->mailjet = $this->getContainer()->get('app.mailjet.client.transactional');
+        $this->messageFactory = $this->getContainer()->get('app.procuration.proxy_message_factory');
+
+        $this->sendMails();
+    }
+
+    private function sendMails($offset = 0, $limit = 100)
```

Comment:
It could, be it will (and should) never be used anywhere else but here.  
I used this parameters.